pkg/proxy/iptables/proxier.go

```diff
@@ -1328,6 +1332,36 @@ func (proxier *Proxier) syncProxyRules() {
 			}
 		}
 
+		// Capture healthCheckNodePorts.
+		if svcInfo.HealthCheckNodePort() != 0 {
+			lps := getLocalPorts("tcp", "healthCheckNodePort", svcInfo.HealthCheckNodePort())
```

Comment:
I don't think we need the OpenPortLocal logic between 1337 - 1352, the health check server binds to that port already which would conflict.

Reply:
Removed, thanks.